BGL/include/CGAL/boost/graph/properties.h

```diff
@@ -227,7 +305,54 @@ void init_halfedge_indices(PolygonMesh& pm, HalfedgeIndexMap hid)
                               >::type >::type() );
 }
 
-} } //end of namespace CGAL::helpers
+} //namespace helpers
+
+
+namespace internal {
+
+template <typename T, typename G>
+typename boost::dynamic_property_map<G,boost::vertex_property_t<T> >::const_type
+add(boost::vertex_property_t<T> prop, const G& g)
```

Comment:
What about renaming it to `add_property/remove_property`. This makes names less generic and limit the risk of ambiguous overloads.